scalarm_client_csharp/Client.cs

```diff
@@ -422,9 +422,8 @@ private ExperimentStatistics HandleExperimentsStatisticsResponse(IRestResponse<E
 
 		public IList<ValuesMap> GetExperimentResults(string experimentId)
 		{
-			var request = new RestRequest("/experiments/{id}/file_with_configurations", Method.GET);
+			var request = new RestRequest("/experiments/{id}/file_with_configurations?error_description=1", Method.GET);
```

Comment:
@Andriu01 - parametry do zapytań w tej bibliotece dodajemy w inny sposób - zobacz w innych miejscach jak to jest robione (nie jest to robione query-stringiem)